tests/test_storage_full_path.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2020 SUSE LLC
```

Comment:
Note: Adding a new test file does 
- not get executed currently in the Jenkins CI (see Jenkinsfile.integration)
- creates a new rook cluster (including all the base OS + CaaSP setup). If you add the test to the test_basic.py file, it will reuse the available rook cluster. That would speed up the whole thing

Reply:
This test cannot use the default-installation of rook/ceph.
Therefore I split up the installation of rook and ceph

Jenkins CI needs to get adopted in a way that e.g. special marked test gets executed as well.

Reply:
Why couldn't this test use the default installation?

After adding the OSD it could remove it and return it to the previous state?

Reply:
Added both tests to test_base.py